Dockerfile

```diff
@@ -4,9 +4,14 @@ MAINTAINER Praekelt Foundation <dev@praekeltfoundation.org>
 # Install nodejs dependencies
 COPY package.json /app/package.json
 WORKDIR /app
-RUN apt-get-install.sh npm && \
-    npm install --production && \
-    apt-get-purge.sh npm
+RUN apt-get-install.sh curl
+
+RUN curl -sL https://deb.nodesource.com/setup_10.x -o nodesource_setup.sh && \
```

Comment:
Is this because we require a newer version of nodejs? The base image has node 4 which we can rely on if that's new enough `praekeltfoundation/vxsandbox:node_4.x`

Reply:
I'll give that a try

Reply:
That one seems to have issues with index files

Reply:
Yeah we shouldn't need to do the `apt-get install npm` if we're using the node in the base image. Should just be able to do the `npm install --production`

Reply:
that makes sense. thanks!